tests/documentLoader.js

```diff
@@ -10,7 +10,8 @@ const loadContext = relativePath => {
 };
 
 const contexts = {
-  "https://www.w3.org/ns/did/v1": loadContext("../contexts/did-v1.jsonld")
+  "https://www.w3.org/ns/did/v1": loadContext("../contexts/did-v1.jsonld"),
+  "https://w3c.github.io/did-core-registries/contexts/did-v1.jsonld": loadContext("../contexts/did-v1.jsonld")
```

Comment:
where is this used?

Reply:
I think this was left over from testing, I will remove.

Reply:
can we resolve this now?

Reply:
Yes, but for some reasons I don't see the resolve button